cmd/webhook.go

```diff
@@ -177,4 +212,6 @@ func init() {
 	webhookCmd.Flags().StringVar(&loglevel, "loglevel", "info", "loglevel to use, one of: debug, info, warn, error, dpanic, panic, fatal")
 	webhookCmd.Flags().DurationVar(&certCheckInterval, "check-interval", 5*time.Minute, "certificate check interval")
 	webhookCmd.Flags().DurationVar(&certLookaheadInterval, "lookahead-interval", crds.LookaheadInterval, "certificate check interval")
+	webhookCmd.Flags().StringVar(&tlsCiphers, "tls-ciphers", "", "comma separated list of tls ciphers to use")
```

Comment:
Can you add examples of how to define ciphers? Please link to the docs where to find the names.

Also please mentions that this is not used for ordering and only controls which TLS 1.0–1.2 cipher suites are enabled. TLS 1.3 ciphers are unaffected by this setting.
And maybe link or add comment to this blog post that explains the mechanics in detail: https://go.dev/blog/tls-cipher-suites

Reply:
Uhh I did not know that the ciphers were not configurable in 1.3. I'll add the info as suggested